.ado/jobs/playground.yml

```diff
@@ -68,9 +68,18 @@ jobs:
           restoreSolution: packages/playground/windows/$(SolutionFile)
           verbosityRestore: Detailed # Options: quiet, normal, detailed
 
+      - task: PowerShell@2
```

Comment:
@chiaramooney : Just to double-check: I thought you created a shared helper yaml for this scenario?
@NickGerleman : I'll leave it up to you to backport that helper and use it, or go with manual expansion here.
Note: Just mentioning to share awareness.

Reply:
It's also very possible I missed a commit. I tried to find the original one, and any followup fixes.